libr/bin/mangling/microsoft_demangle.c

```diff
@@ -145,7 +172,7 @@ static int copy_string(STypeCodeStr *type_code_str, const char *str_for_copy, si
 
 	if (free_space < str_for_copy_len) {
 		int newlen = type_code_str->type_str_len + (str_for_copy_len << 1) + 1;
-		if (newlen < 1) {
+		if (newlen < 1 || newlen > 65536) {
```

Comment:
use UT
```suggestion
		if (newlen < 1 || newlen > UT16_MAX) {
```

Reply:
why this limit

Reply:
@trufae Found frequent OOMs here. IMO 64k is a reasonable limit for any symbol names